packages/cli/src/commands/new/project.ts

```diff
@@ -126,54 +126,58 @@ export const parseConfig = async (
   flags: Partial<ProjectInitializerConfig>,
   boosterVersion: string
 ): Promise<ProjectInitializerConfig> => {
-  if (flags.default) {
+  if (flags.interactive) {
+    const description = await prompter.defaultOrPrompt(
+      flags.description,
+      'What\'s your project description? (default: "")'
+    )
+    const versionPrompt = await prompter.defaultOrPrompt(flags.version, "What's the first version? (default: 0.1.0)")
+    const version = versionPrompt || '0.1.0'
+    const author = await prompter.defaultOrPrompt(flags.author, 'Who\'s the author? (default: "")')
+    const homepage = await prompter.defaultOrPrompt(flags.homepage, 'What\'s the website? (default: "")')
+    const licensePrompt = await prompter.defaultOrPrompt(
+      flags.license,
+      'What license will you be publishing this under? (default: MIT)'
+    )
+    const license = licensePrompt || 'MIT'
+    const repository = await prompter.defaultOrPrompt(
+      flags.repository,
+      'What\'s the URL of the repository? (default: "")'
+    )
+    const providerPackageName = await getProviderPackageName(prompter, flags.providerPackageName)
+
+    return Promise.resolve({
+      projectName: flags.projectName as string,
+      providerPackageName,
+      description,
+      version,
+      author,
+      homepage,
+      license,
+      repository,
+      boosterVersion,
+      installDependencies: flags.installDependencies || false,
+      initializeGit: flags.initializeGit || false,
+      interactive: true,
+    }) 
+  } else {
+    const providerPackageName = await getProviderPackageName(prompter, flags.providerPackageName)
+
     return Promise.resolve({
       projectName: flags.projectName as string,
-      providerPackageName: '@boostercloud/framework-provider-aws',
+      providerPackageName: providerPackageName,
```

Comment:
I think that it would also make sense to add a `--package-manager` flag (and the corresponding interactive step) to set up the package manager, as we now support `npm`, `yarn` and `pnpm`, but this can be done as a separate issue.

Reply:
this makes sense